source/_guides/3-dynamic-rendering.md

```diff
@@ -1,26 +0,0 @@
----
-title: Dynamic rendering
```

Comment:
eliminated this from the turial. mentioning in the rendering chapter but then pointing to reference